README.md

```diff
@@ -68,6 +68,12 @@ Download: [Raspbian Stretch Lite](https://www.raspberrypi.org/downloads/raspbian
 
 Download and install: [Etcher](http://etcher.io)
 
+macOS Catalina: After downloading and attempting to open etcher, you may get a security warning. Go to System Preference -> Security and Privacy -> General and click allow.  If Etcher complains that it cannot write the image, start the program using sudo from the terminal using the command:
```

Comment:
"MacOS" needs to be capitalized. It's likely this affects versions other than Catalina too, so just say "MacOS".
"Etcher" also needs to be capitalized.

Reply:
Catalina introduced breaking changes that Etcher has not yet fixed. The correct format of macOS is lower case first: https://www.apple.com/macos/catalina/.  Happy to change it anyway if you prefer.

Reply:
it's at the start of the sentence though. I see Apple uses lowercase at the start of the sentence too, but that just looks weird to me. How about `In macOS, you may get a security warning when opening Etcher.`? Avoids the capitalization weirdness, and keeps it general enough that other/future versions of macOS are covered too (I think even older versions of macOS can have the same security restrictions enabled that prevent Etcher from running out of the box?)

Reply:
Apple is weird like that. I've revised the pull request.